.ci/macos/dependencies/gource.sh

```
@@ -0,0 +1,56 @@
+# flag during linking and do at least one of the following:
+#    - add LIBDIR to the `DYLD_LIBRARY_PATH' environment variable
+#      during execution
+
+# See any operating system documentation about shared libraries for
+# more information, such as the ld(1) and ld.so(8) manual pages.
+# ----------------------------------------------------------------------
+#  ./install-sh -c -d '/usr/local/include/SDL2'
+#  /usr/bin/install -c -m 644 SDL_image.h '/usr/local/include/SDL2'
+#  ./install-sh -c -d '/usr/local/lib/pkgconfig'
+#  /usr/bin/install -c -m 644 SDL2_image.pc '/usr/local/lib/pkgconfig'
+----------------------------------------------------------------------
```

Shellcheck (0.10.0)

[warning] This flag is used as a command name. Bad line break or missing [ .. ]?

(SC2215)